frontera/tests/test_scrapy.py

```diff
@@ -52,4 +53,8 @@ def test_request_response_converters():
     assert response_converted.meta['test_param'] == 'test_value'
     assert response_converted.url == url
     assert response_converted.status == 200
-    assert response_converted.headers['TestHeader'] == 'Test value'
\ No newline at end of file
+    assert response_converted.headers['TestHeader'] == 'Test value'
+
+    frontier_request = FrontierRequest(url)
+    request_converted = rc.from_frontier(frontier_request)
+    assert frontier_request.url == url
```

Comment:
You check URL here instead of .meta? If you want to check there was no exception, then this line is useless.